pyre/extra_tests/parity_tests/profiled_builtin_raise_keeps_exception.py

```
@@ -0,0 +1,78 @@
+        sys.setprofile(None)
+
+    for name in baseline:
+        assert profiled[name] == baseline[name], (
+            name,
+            baseline[name],
+            profiled[name],
+        )
+
+    # The hook's own errors still win: a profile function that raises replaces
+    # the callee's exception rather than being swallowed by the restore.
+    def angry_hook(frame, event, arg):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `angry_hook`

Add return type annotation: `None`

(ANN202)